test/config/achievement-rule-checker-config.yml

```diff
@@ -52,8 +55,13 @@ achievements:
         requirements:
             - xp: XP
               amount: 10
-    AchievementCanBeAwadedTwice:
+    AchievementCanBeAwardedTwiceAtOnce:
```

Comment:
I think you can just take this out, it doesn't really make sense to configure the service in this manner.